### ISSUE TITLE
Compile issues using crow_all.h in Visual Studio 2019.

### ISSUE THREAD

**Issue by baylf2000** [mrozigor/crow#59]
I'm unable to compile Crow in Visual Studio 2019 using the combined crow_all.h file in an otherwise empty project with just the very simply Crow example.

Using the "Release v0.2" version of crow_all.h I get 31 compile errors. Using a version compile from the latest commit I get three errors as follows:

```
------ Build started: Project: \ConsoleApplication7, Configuration: Release x64 ------
ConsoleApplication7.cpp
Please define _WIN32_WINNT or _WIN32_WINDOWS appropriately. For example:
- add -D_WIN32_WINNT=0x0601 to the compiler command line; or
- add _WIN32_WINNT=0x0601 to your project's Preprocessor Definitions.
Assuming _WIN32_WINNT=0x0601 (i.e. Windows 7 target).
\ConsoleApplication7\crow_all.h(1089,37): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
\ConsoleApplication7\crow_all.h(1338,1): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data
\ConsoleApplication7\crow_all.h(1351,1): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data
\ConsoleApplication7\crow_all.h(1366,1): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data
\ConsoleApplication7\crow_all.h(2777,39): warning C4267: 'argument': conversion from 'size_t' to 'uint32_t', possible loss of data
\ConsoleApplication7\crow_all.h(3043,1): warning C4267: 'return': conversion from 'size_t' to 'int', possible loss of data
\ConsoleApplication7\crow_all.h(3291,1): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
\ConsoleApplication7\crow_all.h(3310,1): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
\ConsoleApplication7\crow_all.h(3312,1): warning C4267: 'argument': conversion from 'size_t' to '_Ty', possible loss of data
        with
        [
            _Ty=int
        ]
\ConsoleApplication7\crow_all.h(3408,1): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data
\ConsoleApplication7\crow_all.h(3536,1): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data
\ConsoleApplication7\crow_all.h(3545,1): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data
\ConsoleApplication7\crow_all.h(3604,1): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
\ConsoleApplication7\crow_all.h(3691,1): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
\ConsoleApplication7\crow_all.h(6240,42): warning C4244: '=': conversion from '__int64' to 'uint16_t', possible loss of data
\ConsoleApplication7\crow_all.h(6247,42): warning C4244: '=': conversion from '__int64' to 'uint16_t', possible loss of data
\ConsoleApplication7\crow_all.h(6254,42): warning C4244: '=': conversion from '__int64' to 'uint16_t', possible loss of data
\ConsoleApplication7\crow_all.h(6263,46): warning C4244: '=': conversion from '__int64' to 'uint16_t', possible loss of data
\ConsoleApplication7\crow_all.h(6356,31): warning C4244: '=': conversion from '__int64' to 'uint16_t', possible loss of data
\ConsoleApplication7\crow_all.h(6496,1): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
\ConsoleApplication7\crow_all.h(7937,1): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
\ConsoleApplication7\crow_all.h(8703,108): warning C4244: 'argument': conversion from '__int64' to 'unsigned int', possible loss of data
\ConsoleApplication7\crow_all.h(8721,109): warning C4244: 'argument': conversion from '__int64' to 'unsigned int', possible loss of data
\ConsoleApplication7\crow_all.h(8739,111): warning C4244: 'argument': conversion from '__int64' to 'unsigned int', possible loss of data
\ConsoleApplication7\crow_all.h(8758,1): warning C4267: 'argument': conversion from 'size_t' to 'unsigned int', possible loss of data
\ConsoleApplication7\crow_all.h(8771,1): warning C4267: 'argument': conversion from 'size_t' to 'unsigned int', possible loss of data
\ConsoleApplication7\crow_all.h(8784,1): warning C4267: 'argument': conversion from 'size_t' to 'unsigned int', possible loss of data
\ConsoleApplication7\crow_all.h(8826,1): warning C4267: '+=': conversion from 'size_t' to 'unsigned int', possible loss of data
\ConsoleApplication7\crow_all.h(8908,1): warning C4267: 'return': conversion from 'size_t' to 'unsigned int', possible loss of data
\ConsoleApplication7\crow_all.h(8954,1): warning C4267: 'argument': conversion from 'size_t' to 'unsigned int', possible loss of data
\ConsoleApplication7\crow_all.h(10513,19): error C2039: 'set_static_file_info': is not a member of 'crow::response'
\ConsoleApplication7\crow_all.h(7638): message : see declaration of 'crow::response'
\ConsoleApplication7\crow_all.h(10508): message : while compiling class template member function 'void crow::Crow<>::run(void)'
\ConsoleApplication7\\ConsoleApplication7.cpp(19): message : see reference to function template instantiation 'void crow::Crow<>::run(void)' being compiled
\ConsoleApplication7\\ConsoleApplication7.cpp(13): message : see reference to class template instantiation 'crow::Crow<>' being compiled
\ConsoleApplication7\crow_all.h(10508,1): error C2039: '__this': is not a member of 'crow::response'
\ConsoleApplication7\crow_all.h(7638): message : see declaration of 'crow::response'
Done building project "\ConsoleApplication7.vcxproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```

Thank you again for your work.

**Comment by The-EDev** [comment]
This seems to be the same as #52 , one of 2 PR's to fix it has been merged (#54), but we're waiting on one more change in the other (#55) to merge it. The issue is fixed by removing all `#if !defined(_WIN32)` lines (in `http_response.h` and `http_connection.h`).

This also enables static file support on windows.

Sorry for the oversight and late response.

Please reopen if the above solution or the PR doesn't work

**Comment by baylf2000** [comment]
Thank you. I made the edits in #55 which fixed the issues and allowed the build to complete.
